### PR TITLE
final portfolio available

### PR DESCRIPTION
**Pull request by rokiasissoko**
Take a look.

### CODE REVIEW

File: portfolio/src/main/webapp/contact.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          <h1> Contact Me </h1>
+            <p> I aspire to take on projects that will impact millions of users globally. Feel free to reach out to me at rssissoko600@gmail.com for any opportunities or additional questions. </p>
+            
+            </br>
+            </br>
+
+            <div class="row">
+                <div class="column">
+                    <a href="https://www.linkedin.com/in/rokiatousissoko/"> 
+                        <img src="/images/linked.png" style="width:75%"/>
```
Comment (tucosh) [inline portfolio/src/main/webapp/contact.html:38]: Are these .png files included somewhere? I'm not seeing them ..

**Review by tucosh — COMMENTED** [review]
Where are the .png files located which you're referencing? Am I just overlooking them?

**Comment by nnooney** [comment]
We copied this code from the deployed site during a 1:1 to setup the GitHub commit flow. Rokia, can you also make a commit containing the images in your site on the branch RokiatouDevelopment and push the branch to GitHub? That will update this PR with the missing .png files and complete this Pull Request.

File: portfolio/src/main/webapp/contact.html (hunk 1 above)
Comment (rokiasissoko) [reply portfolio/src/main/webapp/contact.html:38]: Added the image files

**Comment by rokiasissoko** [comment]
Added all the remaining image files and Week 2 work.

File: portfolio/src/main/webapp/contact.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    <!-- <div class="prideflex">
+      <div class="pridetext">
+      </div>
+    </div> -->
+
+    <div class="contactform">
+
+    <form id="contactform" method="post" action="#">
+      <div class="contanct-title">
```
Comment (tucosh) [inline portfolio/src/main/webapp/contact.html:70]: I think this is a typo..

File: portfolio/src/main/webapp/contact.html [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        </br>
+        </br>
+
+        <label for="email">Message:</label>
+        <textarea name="message" rows="4" cols="50"> </textarea>
+        
+        </br>
+        </br>
+
+        <button type="submit">Submit</button>
```
Comment (tucosh) [inline portfolio/src/main/webapp/contact.html:99]: What's supposed to happen when I fill out the contact form and click Submit? (I tried, it basically redisplayed the page, but something weird happened with the stylesheet I guess, the graphic icons were distorted..)

**Review by tucosh — APPROVED** [review]
It works! LGTM

File: portfolio/src/main/java/com/google/sps/servlets/RandomServlet.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    response.setContentType("text/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Converts a ArrayList<String> instance into a JSON string using manual String concatentation.
+   */
+  private String convertToJson(ArrayList<String> randoFact) {
+    String json = " ";
+    json+= "\"" + randoFact.get((int)Math.floor(Math.random() * randoFact.size())) + "\"";
```
Comment (nnooney) [inline portfolio/src/main/java/com/google/sps/servlets/RandomServlet.java:39]: nit: add a space between json and +=

File: portfolio/src/main/webapp/contact.html [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+  </head>
+  <body> 
+    <div class="top">
+    <header>
+    <img src="images/mintgreen.jpg">
+    </header>
+
+    <nav>
+        <ul>
+            <!-- <li><a href="index.html">Home</a></li> -->
```
Comment (nnooney) [inline portfolio/src/main/webapp/contact.html:17]: Generally it's good practice not to leave commented out code in a file. Can you remove this?

File: portfolio/src/main/webapp/contact.html [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+                    </a>
+                </div>
+                <div class="column">
+                    <a href="https://www.facebook.com/rokiatou.sissoko.9026">
+                        <img src="/images/fb.png" style="width:75%"/>
+                    </a>
+                </div>
+            </div>
+          
+         <!-- <img src="/images/rokiaheadshot.jpeg" /> -->
```
Comment (nnooney) [inline portfolio/src/main/webapp/contact.html:53]: same down here. Can you remove the commented code?

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -7,8 +7,11 @@
     <script src="script.js"></script>
   </head>
   <body> 
+<<<<<<< HEAD
```
Comment (nnooney) [inline portfolio/src/main/webapp/index.html:10]: Whoops looks like there are some merge commits here. Can you resolve these?

**Review by nnooney — DISMISSED** [review]
Looks good! I just left a few minor comments. Could you remove the commented out code? It's good to comment out sections when testing and debugging, but it's harder to read a project when there's a lot of commented out code, so it's good not to check in the commented code to git.

File: portfolio/src/main/webapp/contact.html (hunk 2 above)
Comment (rokiasissoko) [reply portfolio/src/main/webapp/contact.html:70]: Fixed this typo.

File: portfolio/src/main/java/com/google/sps/servlets/RandomServlet.java (hunk 4 above)
Comment (rokiasissoko) [reply portfolio/src/main/java/com/google/sps/servlets/RandomServlet.java:39]: Added the space and deleted all commented out code. Thank you.

File: portfolio/src/main/webapp/contact.html (hunk 3 above)
Comment (rokiasissoko) [reply portfolio/src/main/webapp/contact.html:99]: Working out implementing a confirmation page via Javascript, as of right now it should do nothing.

**Pull request merged**